### ISSUE TITLE
UI Nitpicks

### ISSUE THREAD

**Issue by Darkhost1999** [RPCS3/rpcs3#11392]
As the title suggests this is me being picky and bothered by the UI not being positioned correctly on the screen forcing the User to need to resize or scale the window every single time they are exposed to one of these following screens.

![Screenshot 2022-01-16 231135](https://user-images.githubusercontent.com/60384196/149711702-5e39d376-97ad-44f1-ad5b-f35fd9aae8b5.png)

Starting off: My settings were explicitly set for this test in such a manner that I am not using any windows scaling. 
This is pure default and recommended 100% scaling on a 16:9 50 inch TV screen when using 1280x720p resolution.  This is not a complaint about broken Windows OS scaling as there was no windows scaling involved.
(And yes, I restarted my PC for the settings to take effect)

![Screenshot 2022-01-16 224911](https://user-images.githubusercontent.com/60384196/149711693-3b2844c1-58c3-404e-9714-fd350d6ed25d.png)

Next is the config settings. Any time a user needs to configure settings this screen's save, cancel and apply buttons are constantly blocked by the Windows Taskbar.

![Screenshot 2022-01-16 224941](https://user-images.githubusercontent.com/60384196/149711697-16efec9e-53cb-44fb-8320-80efd6189242.png)

It is the same issue with the Pads screen.

![Screenshot 2022-01-16 225147](https://user-images.githubusercontent.com/60384196/149711701-e1e4b3e6-54dd-4e24-8ffa-cf728dc3a944.png)

And the same issue with the patch creator screen.
I searched every single window in RPCS3 I know about. The UI is perfect aside for this singular issue.

The only time this issue is not present and actually looks like a perfect positioning is when changing the setting from 1280x720 back to 4k without signing out or restarting my PC

![image](https://user-images.githubusercontent.com/60384196/149715614-5289faf6-ce16-428c-9f7e-05c76ba85692.png)

But alas the text, check boxes and everything else are all too small to be read because I need to restart my PC for the 4k 300% scaling to take effect.

My expectation:
I do not expect this issue to be solved quickly as it is a low priority. I expect people not to read anything I said and ask clarifying questions about information I did provide. I also expect that other people will contribute their screenshots saying that they do or do not have this issue.

Keep in mind that this issue is present on Windows 11 OS, with a 50 inch TCL Class 4 series LED 4k UHD HDR Smart Android TV model number 50S434, GTX 1660 Super with default Nvidia control panel settings, and a Ryzen 5 3600.
I expect Linux, Mac, and Windows 10 to potentially have different appearances or results. Although the UI was like this as well with Windows 10 before Windows 11's release. I have only just tolerated this issue regretting every moment of not reporting the issue.

**Comment by Megamouse** [comment]
Closing as duplicate

**Comment by Darkhost1999** [comment]
Strangely I woke up and configure settings screen now fits the screen and I don't even think there were any UI changes. Just rpcs3 automatically updated and bam. Configuring settings. My biggest issue is solved.

I found
![Screenshot_20220117-141123_GitHub.png](https://user-images.githubusercontent.com/60384196/149870405-5fd0a918-43e3-42a6-ae59-bd158c761c50.png)
For a completely understandable answer to why the pads screen is the way it is.

The only remaining window I can't make sense of even with inner layout size. Is the patch creator. It's still highly unnecessarily oversized for a utility window. Where every other window is precisely the size they want/need to be.
